kiwi/repository/apt.py

```diff
@@ -120,7 +121,7 @@ def use_default_location(self):
             os.sep.join([self.manager_base, 'preferences.d'])
         self._write_runtime_config(system_default=True)
 
-    def runtime_config(self):
+    def runtime_config(self) -> dict:
```

Comment:
Please use ```from typing import Dict``` for this type and the other places where this was used. Thanks

Reply:
resolved in latest push